packages/react-examples/src/react/DetailsList/DetailsList.Documents.Example.tsx

```diff
@@ -94,9 +95,11 @@ export class DetailsListDocumentsExample extends React.Component<{}, IDetailsLis
         minWidth: 16,
         maxWidth: 16,
         onColumnClick: this._onColumnClick,
-        onRender: (item: IDocument) => {
-          return <img src={item.iconName} className={classNames.fileIconImg} alt={item.fileType + ' file icon'} />;
-        },
+        onRender: (item: IDocument) => (
+          <TooltipHost content={`${item.fileType} file`} id={`file-type-tooltip-${item.key}`}>
```

Comment:
I don't think you need to manually define `id` here. That's only for if something needs to point to the tooltip as aria-describedby or similar.

Reply:
I assumed id was possibly used for injecting the popup. Will test it out with no id and push if that's the case

Reply:
Indeed it wasn't necessary, changes pushed 👍🏻